pkgs/applications/networking/irc/weechat/default.nix

```diff
@@ -1,8 +1,19 @@
-{ stdenv, fetchurl, ncurses, openssl, perl, python, aspell, gnutls
-, zlib, curl , pkgconfig, libgcrypt, ruby, lua5, tcl, guile
-, pythonPackages, cmake, makeWrapper, libobjc, libiconv
```

Comment:
or add this after the function arguments:
```nix
let
  inherit (pythonPackages) python pycrypto;
in
```

that way the dependency list is still grouped and it doesn't blow-up all-packages.nix too much